script.js

```diff
@@ -16,26 +23,55 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
         },
         data: {
             term: searchTerm,
-            location: searchLocation
+            location: searchLocation,
+            limit: 3,
         }
     }).then(function (result) {
-        //This takes the first three results, can be changed later
-        for (let i = 0; i < 3; i++) {
-            let businessID = result.businesses[i].id
-            let businessName = result.businesses[i].name
-            let businessRating = result.businesses[i].rating
-            let businessPrice = result.businesses[i].price
-            let businessAddress = result.businesses[i].location.display_address
-        }
-    });
+        console.log(result)
+        // result.bussinesses is an array on bussinesses that can be looped through with forEach method ,then for each business object the info are stored in respective array
```

Comment:
here we would create `businessIds=[]`

Reply:
yes instead of creating global variables in our namespace I can change them to local variable in each function to be passes to next function